index.html

```diff
@@ -702,27 +706,120 @@ <h3>Validating an ExposedThingInit</h3>
   <section> <h3>The <dfn>discover()</dfn> method</h3>
     <pre class="idl">
       partial namespace WOT {
-        ThingDiscovery discover(optional ThingFilter filter = null);
+        Promise&lt;ThingDiscoveryProcess&gt; discover(optional ThingFilter filter = null);
```

Comment:
Sorry if I have missed it, but I suppose here we would use the `ExtendedThingFilter`, right?

Reply:
I checked and formally actually made no sense (yet) to extend ThingFilter. We usually simply use prose in the algorithms to describe which properties are used from `filter`. It doesn't matter if someone provides an object that has extra properties, as long as there are the ones that are needed.